script.js

```diff
@@ -0,0 +1,79 @@
+const $ = (selector) => document.querySelector(selector);
+const $$ = (selector) => document.querySelectorAll(selector)
+
+let currentCountries = {}
+
+const getAllCountries = async function () {
+  const response = await fetch('CountriesData.json')
+  return await response.json()
+}
+
+const displayCountries =  function (data) {
+  const countriesGrid = $('.countries-grid');
+  countriesGrid.innerHTML = ''
+
```

Comment:
Please avoid using `innerHTMl`, it is a dangarous method.

This apply to every place in your code using `innerHTMl`